packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2122,15 +2178,12 @@ export const components = (
           },
           "&:hover": {
             color: odysseyTokens.TypographyColorBody,
-            [`& .${tableSortLabelClasses.icon}`]: {
-              opacity: 1,
-            },
           },
           [`&.${tableSortLabelClasses.active}`]: {
             color: odysseyTokens.TypographyColorSubordinate,
             [`& .${tableSortLabelClasses.icon}`]: {
-              opacity: 1,
-              color: "inherit",
+              opacity: "1 !important",
+              color: "inherit !important",
```

Comment:
🤮

We still need `!important`? If so, we should add a comment for why it's here.